salt/minion.py

```diff
@@ -144,10 +144,10 @@ def resolve_dns(opts, fallback=True):
     if (opts.get('file_client', 'remote') == 'local' and
             not opts.get('use_master_when_local', False)):
         check_dns = False
+    # Because I import salt.log below I need to re-import salt.utils here
+    import salt.utils
```

Comment:
I think this should be `salt.utils.network`, and not `salt.utils`. We've moved away from importing `salt.utils` directly in 2018.3 and should be using direct imports to the needed files.